rust/hyper/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "server"
+version = "0.0.0"
+edition = "2021"
+
+[dependencies]
+http = "1.2.0"
+http-body-util = "0.1.2"
+hyper = { version = "1.5.2", features = ["full"] }
+hyper-util = { version = "0.1.10", features = ["full"] }
```

Comment:
is there a way to use the last stable version ? I mean avoid to specify version of this component, but only `hyper`